lib/rucio/daemons/hermes/hermes.py

```diff
@@ -343,6 +343,8 @@ def deliver_emails(
     :returns:                  List of message_id to delete
     """
 
+    smtp_host = config_get("messaging-hermes", "smtp_host")
+    smtp_port = config_get_int("messaging-hermes", "smtp_port", default=25)
```

Comment:
@alexanderrichards 
May be we could add authentication options like username and password to connect to smtp_host and port.
and have smtp `smtp.login` over ssl or starttls or plain